clg/input/generated_common.go

```diff
@@ -0,0 +1,6 @@
+package input
+
+// This file is generated by the CLG generator. Don't edit it manually. The CLG
+// generator is invoked by go generate. For more information about the usage of
+// the CLG generator check https://github.com/xh3b4sd/clggen or have a look at
+// the network package. There is the go generate statement to invoke clggen.
```

Comment:
This comment needs to be fixed in all templates. The `go generate` invocation happens in `clg/clg.go`.